test/alternative_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "spec_helper.rb"
+require "update-alternatives/model/alternative"
+
+describe UpdateAlternatives::Alternative do
+
+  describe ".all" do
```

Comment:
I miss more tests for `.all`, like checking the number of generated alternatives (or even the names, if they are not too many).